schema/viewer.js

```diff
@@ -28,13 +28,17 @@
  * additional fields below.
  */
 
+import CausalityJWT from './causality_jwt';
+import Me from './me';
 import Sales from './sales';
 import { GraphQLObjectType } from 'graphql';
 
 const ViewerType = new GraphQLObjectType({
   name: 'Viewer',
   description: 'A Viewer',
   fields: () => ({
+    causality_jwt: CausalityJWT,
```

Comment:
Ah, I didn't realize that relay didn't support single object endpoints with multiple arguments (thought it only choked on list endpoints), so I imagined this evolving into only supporting those. If that's the case, then this seems fine!